src/thing_shots.c

```diff
@@ -409,7 +418,7 @@ TbBool shot_hit_wall_at(struct Thing *shotng, struct Coord3d *pos)
     {
         if ((blocked_flags & (SlbBloF_WalledX|SlbBloF_WalledY)) != 0)
         {
-            if (shotng->model == ShM_Lizard)
+            if (shot_is_grenade(shotng))
```

Comment:
I don't think this is applicable to the regular grenade too. This is the lizard specific dexterity aiming I believe. Could you check and revert or tell me I'm wrong?

Reply:
I thought the only difference between Lizard and Grenade is that the former is used by a creature's own accord and the latter is not?

Reply:
that's one of the differences. Another difference is that Grenade bounces widely and is untargeted, and Lizard is thrown near a target with how close the  grenade is thrown being dependent on creature dexterity.